tutorials/Google Cloud Hosting.ipynb

```diff
@@ -391,7 +400,7 @@
     "from apscheduler.schedulers.background import BackgroundScheduler\n",
     "from datetime import datetime\n",
     "from flask import Flask\n",
-    "from pyclarifyimport APIClient, DataFrame\n",
+    "from pyclarify import APIClient, DataFrame\n",
```

Comment:
Should this switch to use the new Client?

Reply:
I have decided not to keep updating the notebooks, except the Introduction one.